Mod Configuration Menu/Mods/BG3MCM/ScriptExtender/Lua/Client/IMGUILayer.lua

```diff
@@ -19,7 +19,17 @@
 ---@field private visibilityTriggers table<string, table>
 IMGUILayer = _Class:Create("IMGUILayer", nil, {
     mods = {},
-    visibilityTriggers = {}
+    visibilityTriggers = {},
+    operators = {
+        ["=="] = function(a, b) return a == b end,
+        ["!="] = function(a, b) return a ~= b end,
+        ["<="] = function(a, b) return a <= b end,
+        [">="] = function(a, b) return a >= b end,
+        ["<"] = function(a, b) return a < b end,
+        [">"] = function(a, b) return a > b end,
+        ["isVisible"] = function(a, b) return a == b end
+    },
```

Comment:
Something like this shouldn't be in the `IMGUILayer` class

Reply:
that's why i said it need a visibilityManager class